doc/api-extensions.md

```diff
@@ -2148,3 +2148,13 @@ The Starlark scriptlet is provided to LXD via the new global configuration optio
 Adds support for a `source.wipe` Boolean on the storage pool, indicating
 that LXD should wipe partition headers off the requested disk rather
 than potentially fail due to pre-existing file systems.
+
+## `zfs_block_mode`
+
+This adds support for ZFS block volumes allowing the use of `Btrfs`, `ext4`, and `XFS` file systems on top of ZFS.
```

Comment:
When used in backticks, the rest of the documentation uses `btrfs` and `xfs` (lower case).

@ru-fu what are your thoughts on this?

Perhaps rephrasing to say just:

> This adds support for using ZFS block filesystem volumes allowing the use different filesystems on top of ZFS.

Reply:
We've been using Btrfs and XFS in the docs, since those are clear abbreviations. "ext4" is a bit of a special case there, since it contains a number and must be lowercase. So I would go for "Btrfs, `ext4`, and XFS".

I'm fine with your suggestion as well except that "file system" in general is two words - apart from "ZFS filesystem", which is their terminology. \*argh*
So that would be:
```
This adds support for using ZFS block {spellexception}`filesystem` volumes allowing the use of different file systems on top of ZFS.
```

Reply:
Thanks!